presto-raptor/src/main/java/com/facebook/presto/raptor/metadata/SchemaDao.java

```diff
@@ -50,6 +51,11 @@
             ")")
     void createTableTables();
 
+    @SqlUpdate("ALTER TABLE tables\n" +
+            "  ADD COLUMN table_supports_delta_delete BOOLEAN NOT NULL DEFAULT false,\n" +
```

Comment:
IIRC this column could be null

Reply:
`table_supports_delta_delete BOOLEAN NOT NULL DEFAULT false,\n` in `CREATE TABLE` doesn't seem right. The table property could be null.

Reply:
``table_supports_delta_delete `` is like ``organization_enabled``, ``compaction_enabled``,  we allow null in our tableProperties.
But for meta table in mysql, we don't allow null.